### PR TITLE
Api cleanup

### PR DESCRIPTION
**Pull request by xmonader**
General fixes and cleanup in the ays api for actors, runs

### CODE REVIEW

File: jscockpit/ays_api/views.py [hunk 1]
```diff
@@ -33,12 +33,12 @@ def run_view(run):
...
 
 
-def template_view(t):
+def actor_view(t):
     """
     generate a dict that represent a service from a service object
     """
     template = {
-        'name': t.name,
+        'name': os.path.split(t.path)[-1],
```
Comment (rkhamis) [inline jscockpit/ays_api/views.py]: Why are we not just using the name directly?

File: jscockpit/ays_api/ays.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def listRepositories():
     list all repositorys
     It is handler for GET /ays/repository
     '''
-    repos = j.atyourservice.reposList()
+    repos = j.atyourservice.reposDiscover()
```
Comment (rkhamis) [inline jscockpit/ays_api/ays.py]: THESE ARE NOT EQUAL!
`reposDiscover` scans the fs to find ays repos. `reposList` just grabs whatever is already in redis

File: jscockpit/ays_api/views.py (hunk 1 above)
Comment (xmonader) [inline jscockpit/ays_api/views.py]: I didn't find `name` attribute in the actor object

Comment (rkhamis) [inline jscockpit/ays_api/views.py]: `t.model.name`

**Pull request merged**